### PR TITLE
Fix permission prompt

### PR DESCRIPTION
**Pull request by Lentumunai-Mark**
Entails to resolve permission prompt for Android >= 13

for more context, please refer to the link below
 
Related to this [issue](https://github.com/onaio/fhir-resources/issues/3491).

### CODE REVIEW

File: library/src/main/java/io/ona/kujaku/helpers/PermissionsHelper.java [hunk 1]
```diff
@@ -9,23 +9,33 @@
...
 public class PermissionsHelper {
 
     public static void checkPermissions(String TAG, Context context) {
         if (context instanceof Activity) {
+            List<String> permissions = new ArrayList<>();
+            permissions.add(Manifest.permission.ACCESS_FINE_LOCATION);
+
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.S) {
+                permissions.add(Manifest.permission.WRITE_EXTERNAL_STORAGE);
+            }
```
Comment (dubdabasoduba) [inline library/src/main/java/io/ona/kujaku/helpers/PermissionsHelper.java:27]: @Lentumunai-Mark Do we have tests for this?

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: library/src/main/java/io/ona/kujaku/helpers/PermissionsHelper.java (hunk 1 above)
Comment (Lentumunai-Mark) [reply library/src/main/java/io/ona/kujaku/helpers/PermissionsHelper.java:27]: @dubdabasoduba no existing tests for the class.

Comment (dubdabasoduba) [reply library/src/main/java/io/ona/kujaku/helpers/PermissionsHelper.java:27]: Please create one

File: library/build.gradle [hunk 2]
```diff
@@ -120,9 +120,9 @@ dependencies { configuration ->
     implementation mapboxAnnotationPlugin
 
     // Comment the line below when creating releases - The line is for development of the library & utils
-    //implementation (project(":utils")) {
+    implementation (project(":utils")) {
     // Uncomment the line below when creating releases
-    implementation('io.ona.kujaku:utils:0.10.6-SNAPSHOT') {
+    //implementation('io.ona.kujaku:utils:0.10.7-SNAPSHOT') {
```
Comment (ndegwamartin) [inline library/build.gradle:125]: I think we should revert this change and only have it like this during development. That way, the codebase will always be ready for published snapshots.

Comment (ndegwamartin) [reply library/build.gradle:125]: We can fix later with some conditional logic for better maintainability.

**Review by ndegwamartin — APPROVED** [review]

**Review by dubdabasoduba — APPROVED** [review]
